### ISSUE TITLE
Error loading credentials from netrc file for https://api.github.com/

### ISSUE THREAD

**Issue by rebornix** [ColemanGariety/galileo#13]
Hi,
  I kept getting this error when running this gem. I have checked my account/passwd from ~/_netrc and they are totally correct.

 I'm using Windows 8.1 and ruby 1.9.3 and gem 1.8.28. I'm not a rubyist and I install ruby just because I find this gem is interesting. Let me know if I can provide more info about this error.

**Comment by ColemanGariety** [comment]
Isn't the file `~/.netrc`? Didn't know the `_` worked. Try renaming it?

**Comment by rebornix** [comment]
Yes I suppose it should be .netrc but the _netrc is generated by installation automatically.

I changed it from _netrc to .netrc. However, once I run 'galileo' again, '~/_netrc' would be created again and seems your app would load account/passwd from this file.

**Comment by ColemanGariety** [comment]
@rebornix that's interesting, I don't have an `_netrc` file. What version of the `netrc` gem do you have installed? Maybe I just need to bump it.

**Comment by rebornix** [comment]
I deleted all netric related user files and ran `galileo` again, `$home/_netric` was generated again. My `netric` version is `0.7.7`.

Anything I can do to help get logs or more info?

**Comment by ColemanGariety** [comment]
@rebornix can you do this for me? 

Open a ruby shell.

run:

`n = Netrc.read`
then
`puts n["api.github.com"]`

Let me know what that says.

**Comment by rebornix** [comment]
I got `NameError: uninitialized constant Netrc` after I ran the first command in `irb`. 

BTW, I happened to see this article http://stackoverflow.com/questions/6031214/git-how-to-use-netrc-file-on-windows-to-save-user-and-password . As it says, for windows we need `_netric` instead of `.netric`. Does your code always read `.netric` file directly?

**Comment by ColemanGariety** [comment]
@rebornix oops, I suppose one needs to:

`require "netrc"` first

**Comment by rebornix** [comment]
The result is as below.

```
irb(main):001:0> require "netrc"
=> true
irb(main):002:0> n = Netrc.read
=> #<Netrc:0x20aefe8 @new_item_prefix="", @path="C:/Users/penlv/_netrc", @pre="", @data=[["machine ", "api.github.com",
"\n  login ", "rebornix", "\n  password ", "PASSWORD", "\n"]]>
irb(main):003:0> puts n["api.github.com"]
rebornix
MYPASSWORD
=> nil
```
